v2/pkg/engine/resolve/resolvable_custom_field_renderer_test.go

```diff
@@ -34,903 +34,530 @@ func (t *testCustomResolve) Resolve(ctx *Context, value []byte) ([]byte, error)
 	return t.resolveFunc(ctx, value)
 }
 
+type fieldValueTestCase struct {
+	name                 string
+	input                string
+	fieldValue           Node
+	fieldInfo            *FieldInfo
+	expectedOutput       string
+	expectedFieldName    string
```

Comment:
you could use expectedFieldValue and just compare whole struct instead of separate fields